### DISCUSSION TITLE
Support Maximum Frame Size TLV sending?

### DISCUSSION THREAD

**Discussion by chenkelly** [lldpd/lldpd#465] [Q&A]
Hi
I use the version lldp 1.0.4. 
To tcpdump on Ethernet1, but don't see MFS TLV. 
How to configure and send out LLDP PDU which included the 802.3 MTS TLV?
Thanks very much.
`18: Ethernet1: <BROADCAST,MULTICAST,UP,LOWER_UP> mtu 9100 qdisc pfifo_fast master Bridge state UP mode DEFAULT group default qlen 1000
    link/ether 3c:2c:99:15:aa:60 brd ff:ff:ff:ff:ff:ff
`

**Comment by vincentbernat** [comment]
Yes, it's not here because Linux does not provide a way to retrieve it. https://github.com/lldpd/lldpd/commit/77c4d9d0fdf94d07dd99a08df501c06b8feca9ad

**Reply by chenkelly** [reply]
Thanks.
Can we provide CLI command to configure this MFS value?
Is the method acceptable?
Thanks.

**Reply by vincentbernat** [reply]
Yes, it would be a solution.
